src/manifest-converter.ts

```diff
@@ -92,55 +93,155 @@ export function writeJson(json: any, ...pathPieces: string[]) {
     keywords: bowerJson.keywords,
-    main: (typeof bowerJson.main === 'string') ? bowerJson.main : undefined,
     repository: bowerJson.repository,
-    license: bowerJson.license,
     homepage: bowerJson.homepage,
-    dependencies: <any>{},
-    devDependencies: <any>{},
-    // TODO (fks): Remove these resolutions needed by wct-browser-legacy
-    // https://github.com/Polymer/polymer-modulizer/issues/251
-    resolutions: {
-      'inherits': '2.0.3',
-      'samsam': '1.1.3',
-      'supports-color': '3.1.2',
-      'type-detect': '1.0.0',
-    }
+
+    ...existingPackageJson,
+
+    name,
+    version,
+    flat: true,
+    // TODO(aomarks) We probably want this behind a flag, because switching an
+    // existing package to be publishable could be dangerous. We need this for
+    // the Polymer elements, because the 2.x branches are marked private.
+    private: undefined,
   };
 
-  if (bowerJson.license.includes('polymer.github.io/LICENSE')) {
-    packageJson.license = 'BSD-3-Clause';
-  } else if (!spdxLicenseList.has(bowerJson.license)) {
-    console.warn(
-        `"${bowerJson.license}" is not a valid SPDX license. ` +
-        `You can find a list of valid licenses at https://spdx.org/licenses/`);
+  // TODO (fks): Remove these resolutions needed by wct-browser-legacy
+  // https://github.com/Polymer/polymer-modulizer/issues/251
+  packageJson.resolutions = {
+    ...packageJson.resolutions,
+    'inherits': '2.0.3',
+    'samsam': '1.1.3',
+    'supports-color': '3.1.2',
+    'type-detect': '1.0.0',
+  };
+
+  if (!packageJson.main) {
+    let main;
+    if (typeof bowerJson.main === 'string') {
+      main = bowerJson.main;
+    } else if (Array.isArray(bowerJson.main)) {
+      if (bowerJson.main.length === 1) {
+        main = bowerJson.main[0];
+      } else {
+        // We could potentially be smarter here. Bower configs have loose
+        // semantics around main, and allow one file per filetype. There might
+        // be an HTML file for the main element, and some extra things like a
+        // CSS file. Maybe in that case we should find just the HTML file.
+        //
+        // There could also be multiple HTML files in main, e.g. a repo like
+        // paper-behaviors which contains 3 separate Polymer behaviors. We
+        // currently let that be, so importing the module directly will fail.
+        // We could also generate an index that re-exports all of the symbols
+        // from all of the mains.
+        console.warn(
+            `${bowerJson.name}: Found multiple mains in bower.json, ` +
+            `but package.json must have only one.`);
+      }
+    }
+    if (main && main.endsWith('.html')) {
+      // Assume that the bower main is already a correct relative path to an
+      // HTML file, and that the module equivalent will be in the same directory
+      // but with a JS extension.
+      packageJson.main = replaceHtmlExtensionIfFound(main);
```

Comment:
Add a TODO to use the conversion result to look up the new path.

Reply:
Done.